packages/react-charts/single-packages.config.json

```diff
@@ -4,10 +4,6 @@
         {
             "name": "dist/esm",
             "type": "esm"
-        },
-        {
```

Comment:
Do we not want common js anymore?

Reply:
okay I see they are in the package.json under main.

Reply:
We do, but we don't have to generate `package.json` per each building type. We only need one that references the others. I think we might even consider scanning the src folder instead of the output.